doc/source/theory/pQCD.rst

```diff
@@ -80,23 +85,83 @@ The expression of the pure |QED| and of the mixed |QED| :math:`\otimes` |QCD| sp
+      and the anomalous dimension are then modified using :cite:`Vogt:2004ns`:
+
+        .. math ::
+            & \gamma^{(1)}(N) \to \gamma^{(1)}(N) - \beta_0 k \gamma^{(0)} \\
+            & \gamma^{(2)}(N) \to \gamma^{(2)}(N) - 2 \beta_0 k \gamma^{(1)} - ( \beta_1 k - \beta_0^2 k^2) \gamma^{(0)} \\
+            & \gamma^{(3)}(N) \to \gamma^{(3)}(N) - 3 \beta_0 k \gamma^{(2)} - ( 2 \beta_1 k - 3 \beta_0^2 k^2) \gamma^{(1)} - (\beta_2 k - \frac{5}{2} \beta_1 \beta_0 k^2 + \beta_0^3 k^3) \gamma^{(0)} \\
+            & k = \ln(\mu_F^2/\mu_R^2)
+
+      This procedure corresponds to Eq. (3.32) of :cite:`AbdulKhalek:2019ihb`, and we recommend to use it along with
+      ``ModEv='iterate-exact'`` in order to be in agreement with the treatment of the evolution integral expansion.
+
+
+    * In ``ModSV='expanded'`` the |EKO| is multiplied by an additional kernel, such that
+      the scale variation is applied to the whole |PDF| set:
+
+        .. math ::
+            & \tilde{\mathbf{E}}(a_s \leftarrow a_s^0) = \tilde{\mathbf{K}}(a_s) \tilde{\mathbf{E}}(a_s \leftarrow a_s^0) \\
+            & \tilde{\mathbf{K}}(a_s) = 1 - k \gamma + \frac{1}{2} k^2 \left ( \gamma^{2} - \beta \frac{\partial \gamma}{\partial a_s} \right ) + \frac{1}{6} k^3 \left [ - \beta \frac{\partial}{\partial a_s} \left( \beta \frac{\partial \gamma}{\partial a_s} \right) + 3 \beta \frac{\partial \gamma}{\partial a_s} \gamma - \gamma^3 \right ] + \mathcal{O}(k^4)
+
+      where scale variation kernel is expanded consistently order by order in :math:`a_s`,
+      leading to:
+
+        .. math ::
+            \tilde{\mathbf{K}}(a_s) \approx & 1 - a_s k \gamma^{(0)} + a_s^2 \left [ - k \gamma^{(1)} + \frac{1}{2} k^2 \gamma^{(0)} (\beta_0 + \gamma^{(0)}) \right ] \\
+            & + a_s^3 \left [ -k \gamma^{(2)} + \frac{1}{2} k^2 \left(\beta_1 \gamma^{(0)} + 2 \gamma^{(1)} (\beta_0 + \gamma^{(0)} ) \right) - \frac{1}{6} k^3 \gamma^{(0)} \left(2 \beta_0^2 + 3 \beta_0 \gamma^{(0)}+\gamma^{(0),2} \right) \right] + \mathcal{O}(a^4)
+
+
+      In this way the dependence of the |EKO| on :math:`k` is factorized outside the unvaried evolution kernel.
+      This procedure is repeated for each different flavor patch present in the evolution path.
+      It corresponds to Eq. (3.35) of :cite:`AbdulKhalek:2019ihb`, and we recommend to use it along with
+      ``ModEv='truncated'`` in order to keep consistency with the evolution integral expansion.
+
+
+By construction, the corrections of the order :math:`\mathcal{O}(k^n)` will appear
+at the order :math:`n` in the expansion :math:`a_s`.
+This happens because :math:`\beta \approx \mathcal{O}(a_s^2)`, :math:`\gamma \approx \mathcal{O}(a_s)`
+and the contribution proportional to :math:`\mathcal{O}(k^n)` is originated
+by the `n-th` derivative in :math:`\gamma` :cite:`AbdulKhalek:2019ihb`.
 
+Furthermore the distance between the varied |EKO| and the unvaried one will decrease increasing
```

Comment:
Can you find an easy rewording such that the sentence becomes more clean? (I find "decrease increasing" confusing :upside_down_face: )